x-pack/plugins/security_solution/public/overview/pages/detection_response.tsx

```diff
@@ -17,20 +17,41 @@ import { useSourcererDataView } from '../../common/containers/sourcerer';
 import { useAlertsPrivileges } from '../../detections/containers/detection_engine/alerts/use_alerts_privileges';
 import { HeaderPage } from '../../common/components/header_page';
 import { useShallowEqualSelector } from '../../common/hooks/use_selector';
-import { DETECTION_RESPONSE_TITLE, UPDATED, UPDATING } from './translations';
+import { DETECTION_RESPONSE_TITLE, UPDATED, UPDATING, VIEW_ALERTS } from './translations';
 import { inputsSelectors } from '../../common/store/selectors';
+import { AlertsByStatus } from '../components/detection_response/alerts_by_status';
+import { useUserInfo } from '../../detections/components/user_info';
 import { LandingPageComponent } from '../../common/components/landing_page';
+import { DETECTION_RESPONSE_ALERTS_BY_STATUS_ID } from '../components/detection_response/alerts_by_status/utils';
+import { getDetectionEngineUrl, useFormatUrl } from '../../common/components/link_to';
+import { useKibana } from '../../common/lib/kibana/kibana_react';
+import { APP_UI_ID } from '../../../common/constants';
 
 const DetectionResponseComponent = () => {
   const getGlobalQuery = useMemo(() => inputsSelectors.globalQuery(), []);
   const { indicesExist, indexPattern, loading: isSourcererLoading } = useSourcererDataView();
+  const { loading: loadingSignalIndex, signalIndexName } = useUserInfo();
   const [updatedAt, setUpdatedAt] = useState(Date.now());
   // TODO: link queries with global time queries
   // const { to, from, deleteQuery, setQuery, isInitializing } = useGlobalTime();
 
   const queriesLoading: boolean = useShallowEqualSelector(
     (state) => !!getGlobalQuery(state).find((query) => query.loading)
   );
+  const kibana = useKibana();
+  const { navigateToApp } = kibana.services.application;
+  const { formatUrl, search: urlSearch } = useFormatUrl(SecurityPageName.alerts);
+
+  const goToAlerts = useCallback(
+    (ev) => {
+      ev.preventDefault();
+      navigateToApp(APP_UI_ID, {
+        deepLinkId: SecurityPageName.alerts,
+        path: getDetectionEngineUrl(urlSearch),
+      });
+    },
+    [navigateToApp, urlSearch]
+  );
```

Comment:
NIT:we can use the `const { navigateTo, getAppUrl } = useNavigation()` in our common kibana hooks, so we don't have to import `useKibana`, `APP_UI_ID`...
(this code should be moved from here to the component that does the navigation)